addons/l10n_it_edi/tests/test_edi_import.py

```diff
@@ -180,14 +181,15 @@ def mock_commit(self):
               patch.object(sql_db.Cursor, "commit", mock_commit),
               tools.mute_logger("odoo.addons.l10n_it_edi.models.account_move")):
             for dummy in range(2):
-                self.env['account.move']._l10n_it_edi_process_downloads({
+                processed = self.env['account.move']._l10n_it_edi_process_downloads({
                     '999999999': {
                         'filename': filename,
                         'file': self.fake_test_content,
                         'key': str(uuid.uuid4()),
                     }},
                     proxy_user,
                 )
+                proxy_acks += processed['proxy_acks']
```

Comment:
```suggestion
                # The Proxy ACK must be sent in both cases of import success and failure.
                self.assertEqual(processed['proxy_acks'], ['999999999'])
```
We can directly do the check here and remove the rest, what do you think?

Reply:
I guess it is a good idea, I've implemented the requested changes.
thank you for the review :)

Reply:
Thank you for the fix!